frontend/src/hooks/useSearchParamUpdater.ts

```diff
@@ -12,7 +11,10 @@ export function useSearchParamUpdater() {
     queryParamValue: string | Set<string>,
     key: string,
   ) => {
-    const params = new URLSearchParams(searchParams || {});
+    // TODO: Next's useSearchParams was causing issues. document.location.search
+    // seems to work better when calling from the form submit. Some follow up work
+    // to investigate if URLSearchParams(useSearchParams()) can still work.
```

Comment:
Is there an issue that can be added to this comment?

Reply:
added issue #s for the ones you commented on - this second one might not be worth it to reinvistage (I think it might not work for us) - but we can keep a ticket for it at least for now.